lib/ahoy/ahoy.js

```diff
@@ -82,8 +82,8 @@ export default class Ahoy {
   }
 
   initConnection = async () => {
-    this.hasInternetAccess = await NetInfo.isConnected.fetch();
-    NetInfo.isConnected.addEventListener('connectionChange', this.handleNetInfoConnectionChange);
+    let req = await fetch('https://www.google.com');
```

Comment:
I think change native connection change state on fetch to site - that a bad idea. if the site is not available, but this connection is active, we will get an error.

Reply:
I will use a native solution in a next PR